<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpPage</class>
 <widget class="QDialog" name="SignUpPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign Up</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
  </property>
  <widget class="QLabel" name="label_birthday">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>251</y>
     <width>87</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
   </property>
   <property name="text">
    <string>Birthday</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>289</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
   </property>
   <property name="text">
    <string>Country</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_country">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>321</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="pushButton_createAccount">
      <property name="styleSheet">
       <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
      </property>
      <property name="text">
       <string>Create  Account</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_back">
      <property name="styleSheet">
       <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>332</y>
     <width>255</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_gender">
      <property name="styleSheet">
       <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_female">
      <property name="styleSheet">
       <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
      </property>
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_male">
      <property name="styleSheet">
       <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
      </property>
      <property name="text">
       <string>Male</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>621</width>
     <height>170</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_firstname">
      <property name="styleSheet">
       <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_firstname"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_lastname">
      <property name="styleSheet">
       <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_lastname"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_username">
      <property name="styleSheet">
       <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_username"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_password">
      <property name="styleSheet">
       <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_password"/>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_passwordRequirements">
      <property name="styleSheet">
       <string notr="true">font-size:15px;
font-weight:none;
</string>
      </property>
      <property name="text">
       <string>Use at least 6 charaster. Must include a number and a capital letter </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>251</y>
     <width>521</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_moth">
      <property name="styleSheet">
       <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
      </property>
      <property name="text">
       <string>Month</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_months"/>
    </item>
    <item>
     <widget class="QLabel" name="label_date">
      <property name="styleSheet">
       <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_date"/>
    </item>
    <item>
     <widget class="QLabel" name="label_year">
      <property name="styleSheet">
       <string notr="true">font-family:sans-serif;
font-size:20px;
font-weight:bold;
color:#545454;</string>
      </property>
      <property name="text">
       <string>Year</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_year"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
